dcm4che-net/src/test/java/org/dcm4che3/net/DeviceTest.java

```diff
@@ -37,50 +37,88 @@
  * ***** END LICENSE BLOCK ***** */
 package org.dcm4che3.net;
 
+import static org.hamcrest.Matchers.containsInAnyOrder;
 import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertThat;
 
+import java.util.Arrays;
 import java.util.List;
 
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
 
 /**
  * @author Gunter Zeilinger <gunterze@gmail.com>
  *
  */
 public class DeviceTest {
 
-    /**
-     * Test method for {@link org.dcm4che3.net.Device#reconfigure(org.dcm4che3.net.Device)}.
-     */
+    private static final String AE_TITLE = "MyAeTitle";
+    private static final String FIRST_ALIAS_AE_TITLE = "AliasAeTitle";
+    private static final String SECOND_ALIAS_AE_TITLE = "AnotherAliasAeTitle";
+
+    @Rule
+    public TemporaryFolder temporaryFolder = new TemporaryFolder();
+
+    private ApplicationEntity ae;
+
     @Test
     public void testReconfigure() throws Exception {
         Device d1 = createDevice("test", "AET1");
         Device d2 = createDevice("test", "AET2");
         d2.setOlockHash("I.hash.you.not");
-        d2.setStorageVersion(35);
-        
+                                                                d2.setStorageVersion(35);
+
         d1.reconfigure(d2);
         ApplicationEntity ae = d1.getApplicationEntity("AET2");
         assertNotNull(ae);
         List<Connection> conns = ae.getConnections();
         assertEquals(1, conns.size());
-        
-       assertThat("Olock Hash", d1.getOlockHash(), equalTo("I.hash.you.not"));
-       assertThat("Storage version", d1.getStorageVersion(), equalTo(35L));
-       
+
+        assertThat("Olock Hash", d1.getOlockHash(), equalTo("I.hash.you.not"));
```

Comment:
Now I see what is going on here...we have our IDEs setup for tab being 3 spaces, but original file was 4 spaces per tab...I'm not sure what is the best approach here., but I see that in master even Device class uses 4 so I guess better to conform to their rule here.

Reply:
not sure we have 4 spaces for tab